config/routes.rb

```diff
@@ -150,6 +150,9 @@
         end
         get :show_interested_other_positions, to: 'applicants#show_interested_other_positions'
         get :show_unflagged_applicants, to: 'applicants#show_unflagged_applicants'
+        get :show_unlogged_applicants, to: 'applicants#show_unlogged_applicants'
+        patch :show_unlogged_applicants, to: 'applicants#log_single_applicant'
+        post :show_unlogged_applicants, to: 'applicants#log_all_applicants'
```

Comment:
Maybe there is an easier to way to handle two forms in a single view without using different method?